tensorflow/compiler/xla/service/cpu/simple_orc_jit.cc

```diff
@@ -102,9 +102,15 @@ llvm::StringRef GetHostCpuName() {
 
 CompilerFunctor::VectorIntrinsics GetAvailableIntrinsics() {
   CompilerFunctor::VectorIntrinsics intrinsics;
-  intrinsics.sse_intrinsics = (&__xla_cpu_runtime_ExpV4F32SSE != nullptr);
-  intrinsics.avx_intrinsics = (&__xla_cpu_runtime_ExpV8F32AVX != nullptr);
-  intrinsics.neon_intrinsics = (&__xla_cpu_runtime_ExpV4F32NEON != nullptr);
+#ifdef __SSE4_1__
+  intrinsics.sse_intrinsics = &__xla_cpu_runtime_ExpV4F32SSE;
```

Comment:
These are `bool` fields; you need to do:

```
#ifdef __SSE_4_1__
  intrinsics.sse_intrinsics = true;
#else
  intrinsics.sse_intrinsics = false;
#endif
```

etc.

Reply:
got it. updated now.